common/djangoapps/third_party_auth/settings.py

```diff
@@ -85,3 +84,5 @@ def apply_settings(django_settings):
         'social.apps.django_app.context_processors.backends',
         'social.apps.django_app.context_processors.login_redirect',
     )
+
+    django_settings.OPTIONAL_USER_FIELDS = ['first_name', 'last_name', 'fullname']
```

Comment:
@bradenmacdonald typo - should be `EXTRA_USER_FIELDS`. `EXTRA_USER_FIELDS` is used in our friend `create_user` to pass more data to user creation strategy. The reason for this is that normally, only username and email required, so `USER_FIELDS` contain only `username` and  `email`, and only those fields are passed to strategy.create_user. With autoprovision we want to pass as much data is we can, but still not make it required.

Actually, looks like I forgot that `get(name)` with single parameter acts as `get(name, None)`, so those fields could be safely added to `USER_FIELDS`. So I'm going to remove it.